ecoflow_exporter.py

```diff
@@ -275,7 +275,7 @@ def process_payload(self, params):
         log.debug(f"Processing params: {params}")
         for ecoflow_payload_key in params.keys():
             ecoflow_payload_value = params[ecoflow_payload_key]
```

Comment:
m.b better to use `items()` here?
```suggestion
        for ecoflow_payload_key, ecoflow_payload_value in params.items():
```